addons/stock/models/stock_move_line.py

```diff
@@ -79,7 +79,7 @@ def _check_lot_product(self):
     def _check_same_company_id(self):
         for line in self:
             if line.move_id and line.move_id.company_id != line.company_id:
-                raise ValidationError(_('The move line a a different company than its move'))
+                raise ValidationError(_('The move line has a different company than its move'))
```

Comment:
Could you move this diff in another commit ("fix typo ...")

Reply:
It's done. :)